### ISSUE TITLE
Op info test for `masked.softmax .. masked.var`

### ISSUE THREAD

**Issue by qihqi** [pytorch/xla#7511]
## Fix the Op info test for `masked.softmax .. masked.var`

1. Find the lines 173 to 177 of [test_ops.py](test/test_ops.py) and remove
   `masked.softmax` .. `masked.var` from `skip_list`
2. Run op_info test with `pytest test/test_ops.py`
3. Fix the failure.

Please refer to 
[this guide](https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/docs/fixing_op_info_test.md) as guide to fix:

Also refer to these PRs:
* https://github.com/pytorch/xla/pull/7343
* https://github.com/pytorch/xla/pull/7312
* https://github.com/pytorch/xla/pull/7325

**Comment by ManfeiBai** [comment]
hi, @tqfan28, thanks for contribution, this would be the first issue to add torch_xla2 ops

**Comment by tqfan28** [comment]
Thanks, Manfei, please assign it to me.

**Comment by tqfan28** [comment]
Created pull request: https://github.com/pytorch/xla/pull/7944
"masked.softmin", "masked.std", "masked.sum", "masked.var" already pass the test.
Add a minor fix for "masked.softmax"
